gui/createGradebookFinalGradeFormulas.js

```diff
@@ -21,7 +21,7 @@ new QSGradebookIterator(function() {
 
     this.click("Edit it anyways");
     this.click("Add Formula");
-    $("tr.category:contains(Assignment) input").click();
+    $("tr.category:contains(Classwork) input").click();
```

Comment:
Yeah... Assignment seems fine. But, I guess it could vary. Might as well leave the default there though